app/client/stylesheets/variables.scss

```diff
@@ -152,7 +152,7 @@ $app-genre-margin-top: 5.417rem;
 $app-genre-margin-bottom: 1.333rem;
 $app-container-width: 21.3rem;
 $app-container-width-large: $app-container-width * 3;
-$app-container-height: 32rem;
+$app-container-height: 33rem;
```

Comment:
There wasn't sufficient padding/whitespacec available for the rating bar. Adding padding to the rating bar wouldn't have given it more space so I increased the height of the app-container a touch.